rkviewer/forms.py

```diff
@@ -236,9 +236,8 @@ def InitAndGetSizer(self) -> wx.GridSizer:
         # Ensure the input field takes up some percentage of width
         # Note that we might want to adjust this when scrollbars are displayed, but only in case
         # there is not enough width to display everything
-        width = self.GetSize()[0]
-        right_width = (width - VGAP * 3 - MORE_LEFT_PADDING - MORE_RIGHT_PADDING -
-                       self._info_length) * .7
+        right_width = (edit_panel_width - VGAP * 3 - MORE_LEFT_PADDING - MORE_RIGHT_PADDING -
+                       self._info_length) * .5
```

Comment:
set this value (0.5) to change the ratio of text input width to edit panel width